src/main/sensors/gyro.c

```diff
@@ -552,7 +646,22 @@ bool isGyroSensorCalibrationComplete(const gyroSensor_t *gyroSensor)
 
 bool isGyroCalibrationComplete(void)
 {
+#ifdef USE_DUAL_GYRO
```

Comment:
Code could be easier if `gyroToUse` was a bitmask. Something to consider for more than 2 gyros :-)

Reply:
yeah, that was @martinbudden 's bit of code.  You're right but I don't see the need for more than 2 any time soon.